src/org/traccar/Context.java

```diff
@@ -349,24 +354,7 @@ public static void init(String[] arguments) throws Exception {
         }
 
         if (config.getBoolean("geolocation.enable")) {
-            String type = config.getString("geolocation.type", "mozilla");
-            String url = config.getString("geolocation.url");
-            String key = config.getString("geolocation.key");
-
-            switch (type) {
-                case "google":
-                    geolocationProvider = new GoogleGeolocationProvider(key);
-                    break;
-                case "opencellid":
-                    geolocationProvider = new OpenCellIdGeolocationProvider(key);
-                    break;
-                case "unwired":
-                    geolocationProvider = new UnwiredGeolocationProvider(url, key);
-                    break;
-                default:
-                    geolocationProvider = new MozillaGeolocationProvider(key);
-                    break;
-            }
+            initLocationResolutionSystem();
```

Comment:
Please rename to `initGeolocationSystem` or even better `initGeolocationModule`.

Reply:
done